client/src/components/EditProfile/ProfilePhoto/UploadPhoto.tsx

```diff
@@ -15,41 +15,34 @@ const UploadPhoto: React.FC = (): JSX.Element => {
     }
   };
 
+  const avatarSrc = selectedImage && URL.createObjectURL(selectedImage);
+
   return (
-    <Box className={classes.root}>
-      <form>
-        <Typography variant="h5" className={classes.typography}>
-          Profile Photo
-        </Typography>
-        <input
-          accept="image/*"
-          className={classes.input}
-          id="contained-button-file"
-          multiple
-          type="file"
-          onChange={imageChange}
-          required
-        />
-        <label htmlFor="outlined-button-file">
-          {selectedImage ? (
-            <Avatar src={URL.createObjectURL(selectedImage)} alt="Thumb" className={classes.avatar} />
-          ) : (
-            <Avatar className={classes.avatar} />
-          )}
-          <Typography variant="subtitle1" className={classes.typography}>
-            Be sure to use a photo that clearly shows your face
-          </Typography>
-          <Button variant="outlined" size="large" color="primary" component="span" className={classes.button}>
-            Upload a file from your device
-          </Button>
-          <Box>
-            <IconButton aria-label="delete" className={classes.iconButton}>
-              <DeleteIcon fontSize="large" />
-              <Typography variant="subtitle1">Delete photo</Typography>
-            </IconButton>
-          </Box>
-        </label>
-      </form>
+    <Box className={classes.root} component="form">
+      <Typography variant="h5" className={classes.typography}>
+        Profile Photo
+      </Typography>
+      <input
+        accept="image/*"
+        className={classes.input}
+        id="button"
+        multiple
+        type="file"
+        onChange={imageChange}
+        required
+      />
+      <Avatar src={avatarSrc} alt="Profile photo/avatar/thumbnail" className={classes.avatar} />
```

Comment:
Ah, sorry for the confusion here but I use the `/` character to indicate interchangeable keywords. In this case, I was suggesting to use either one of "Profile photo", "Profile avatar" or "Profile thumbnail". Please address this 🙏🏼